ddl/ddl_algorithm.go

```diff
@@ -25,6 +25,7 @@ import (
 // because we never block the DML but costs some time to backfill the index data)
 // See https://dev.mysql.com/doc/refman/8.0/en/alter-table.html#alter-table-performance.
 type AlterAlgorithm struct {
+	// supported should store algorithms in the order 'INSTANT, INPLACE, COPY'
```

Comment:
```suggestion
	// supported MUST store algorithms in the order 'INSTANT, INPLACE, COPY'
```
I guess we must ensure the order, right? It's better to emphasize it in the comment.

Reply:
Yes, it is true.